sql/sampledata.log

```diff
@@ -0,0 +1,6 @@
+/*C4*/SET SCHEMA PUBLIC
```

Comment:
Please gitignore this "*.log" file or any developer that runs the build or samples will modify it needlessly. HSQL will regenerate it if it is missing without any problems.